dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/gateway/AkkaHttpServiceGateway.scala

```diff
@@ -116,6 +121,20 @@ class AkkaHttpServiceGateway(lifecycle: ApplicationLifecycle, config: ServiceGat
     )
   }
 
+  private def createRequestHeader(request: HttpRequest): RequestHeader = {
```

Comment:
Maybe we should add some comments to these since it is not that obvious why they exist.